core/src/test/resources/audit/data_element_create.json

```diff
@@ -0,0 +1,8 @@
+{
+  "createdAt": "2018-02-05T17:19:27.106",
```

Comment:
I have tested a create change and I have received entity as value 
{
  "createdAt": "2018-02-06T12:51:35.982",
  "createdBy": "admin",
  "klass": "org.hisp.dhis.dataelement.DataElement",
  "uid": "Qj53klmSuiW",
  "type": "CREATE",
  "value": {
    "name": "AA",
    "created": "2018-02-06T12:51:35.833",
    "lastUpdated": "2018-02-06T12:51:35.833",
    "externalAccess": false,
    "publicAccess": "rw------",
    "user": {
      "name": "John Traore",
      "created": "2013-04-18T17:15:08.407",
      "lastUpdated": "2017-06-02T13:04:58.319",
      "externalAccess": false,
      "displayName": "John Traore",
      "id": "xE7jOejl9FI"
    },
    "displayName": "AA",
    "lastUpdatedBy": {
      "id": "xE7jOejl9FI",
      "name": "John Traore"
    },
    "shortName": "AA",
    "displayShortName": "AA",
    "dimensionItemType": "DATA_ELEMENT",
    "aggregationType": "SUM",
    "valueType": "NUMBER",
    "displayFormName": "AA",
    "domainType": "AGGREGATE",
    "zeroIsSignificant": false,
    "optionSetValue": false,
    "dimensionItem": "Qj53klmSuiW",
    "id": "Qj53klmSuiW",
    "categoryCombo": {
      "name": "default",
      "created": "2011-12-24T12:24:25.203",
      "lastUpdated": "2016-04-12T20:37:49.126",
      "externalAccess": false,
      "displayName": "default",
      "id": "p0KPaWEg3cf"
    }
  }
}